pom.xml

```diff
@@ -91,6 +92,12 @@
         <version>${websocket.version}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
```

Comment:
Let's add comment that it's just for websocket to stop polluting the output with warnings.